server/src/main/java/com/paypal/selion/utils/ServletHelper.java

```diff
@@ -115,124 +118,26 @@ public static void respondAsHtmlUsingJsonAndTemplateWithHttpStatus(HttpServletRe
-        writer.write("        </li></ul></form>");
-        writer.write("<div id='footer'>");
-        writer.write("<p>Created by the SeLion Project</p>");
-        writer.write("</div>");
-        writer.write("</div> <img id='bottom' src='/grid/resources/form/bottom.png' alt=''>");
-        writer.write("</body>");
-        writer.write("</html>");
-    }
-
-    /**
-     * Utility method to display footer to a HTML page which contains Home and Logout link
-     * 
-     * @param writer
-     *            The {@link PrintWriter} object that corresponds to a response
-     */
-    public static void displayFooter(PrintWriter writer) {
-        writer.write("<div id='footer'>");
-        writer.write("<a href='/grid/admin/" + LoginServlet.class.getSimpleName() + "?form_id=login'>Go To Home</a>");
-        writer.write("&nbsp;&nbsp;&nbsp;&nbsp;");
-        writer.write("<a href='/grid/admin/" + LoginServlet.class.getSimpleName() + "?logout=true'>Logout</a>");
-        writer.write("<p>Created by the SeLion Project</p>");
-        writer.write("</div>");
-        writer.write("</div> <img id='bottom' src='/grid/resources/form/bottom.png' alt=''>");
+    public static void displayMessageOnRedirect(PrintWriter writer, String reDirectMessage) throws IOException {
+        respondWithTemplate(writer, MESSAGE_RESOURCE_PAGE_FILE, reDirectMessage);
     }
 
     /**
-     * Utility method to display header to a HTML page
+     * Utility method to load the template from resource, insert the arguments in template and write to writer
      * 
      * @param writer
      *            The {@link PrintWriter} object that corresponds to a response
+     * @param page
+     *            HTML template page for response
+     * @param args
+     *            Data which will be inserted in to the template
+     * @throws IOException
      */
-    public static void displayHeader(PrintWriter writer) {
-        writer.write("<!DOCTYPE html PUBLIC \"-//W3C//DTD XHTML 1.0 Transitional//EN\" \"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd\">");
-        writer.write("<html xmlns=\"http://www.w3.org/1999/xhtml\">");
-        writer.write("<head>");
-        writer.write("<meta http-equiv='Content-Type' content='text/html; charset=UTF-8'>");
-        writer.write("<title>Grid Management Console</title>");
-        writer.write("<link rel='stylesheet' type='text/css' href='/grid/resources/form/view.css' media='all' >");
-        writer.write("<script type='text/javascript' src='/grid/resources/form/view.js'></script>");
-        writer.write("</head>");
-        writer.write("<body id='main_body'>");
-        writer.write("<img id='top' src='/grid/resources/form/top.png' alt='' >");
+    public static void respondWithTemplate(PrintWriter writer, String page, Object... args)
+            throws IOException {
+        String template = IOUtils.toString(LoginServlet.class.getResourceAsStream(page), "UTF-8");
```

Comment:
`LoginServlet.class` should not be used here. I will fix this and update the branch